sui/its-example.js

```diff
@@ -260,6 +318,7 @@ async function deployToken(keypair, client, contracts, args, options) {
             TokenId: tokenId,
             origin: options.origin,
         },
+        tokenManagerMode: options.tokenManagerMode || 'lock_unlock', // default to lock_unlock
```

Comment:
I am working on another PR that makes `tokenManagerMode` a mandatory param, since in `sui/its link-coin` it's mandatory for consistency. I'll make sure I update this line as well afterward, since the default won't be needed anymore.